doc/guides/conflicts.md

```diff
@@ -0,0 +1,110 @@
+## Conflict resolution
+
+## Prerequisites:
+
+- GraphQL server with resolvers
+- Database or any other form of data storage that can cause data conflicts
+
+## Pluggable conflict resolution
+
+Pluggable conflict resolution is an concept that allows developers to define handling for collisions between data sent to resolver
+and data that is currently stored in the server.
```

Comment:
I know markdown will fix it, but it should not have a breaking line

Reply:
@danielpassos I wouldn't be too worried about this because usually it makes the file more readable inside an editor 🙂

In fact a lot of projects actually enforce max line lengths in markdown files and force you to do this. I would personally never go that far and enforce it but I so generally see this as a good thing!